posthog/queries/query_date_range.py

```diff
@@ -147,7 +147,7 @@ def date_to(self) -> Tuple[str, Dict]:
         date_to_query = self.date_to_clause
         date_to = self.date_to_param
 
-        if not self.is_hourly(self._filter._date_to):
+        if not self.is_hourly(self._filter._date_to) and not self._filter.use_explicit_dates:
```

Comment:
I think we should not add another param to the filter especially because this parameter isn't being passed in from the client. We can make `use_explicit_date` a parameter for this class QueryDateRange and then also have a property field `use_explicit_field` that looks at both the is_hourly logic along with the override param

Reply:
that means passing it in to the funnel somehow, since the experiment only controls the filter right now, which it passes it on to the funnel query